src/mixins/filing-template-mixin.ts

```diff
@@ -144,8 +144,7 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
         amalgamatingBusinesses: this.getAmalgamatingBusinesses,
         type: this.getAmalgamationType,
         nameRequest: {
-          legalType: this.getEntityType,
-          correctNameOption: this.getCorrectNameOption
```

Comment:
Not needed as it's set on lines 178+ below.

This is a the "decoy" anti-pattern: it looks like it's doing something but it's really not.